### ISSUE TITLE
IBM AppScan parser does not parse out the cve nor cwe codes

### ISSUE THREAD

**Issue by mhanner** [DefectDojo/django-DefectDojo#6479]
**Bug description**
The IBM AppScan parser does not parse the cve or cwe codes from the scan file.

I believe the root problem is in fetch_issue_types()  - "cve" is not in issue_type, it is in the item element.

The cve and cwe are parsed using the attached patch.  I tested it on all the sample appscan reports included in DefectDojo as well as other reports. 

[0001-appscan-parser-does-not-parse-cve.patch.txt](https://github.com/DefectDojo/django-DefectDojo/files/9005749/0001-appscan-parser-does-not-parse-cve.patch.txt)

**Steps to reproduce**
1. Import an AppScan scan file
2. View fidings 
3. Notice that none of the findings have a CWE or CVE (Vulnerability id)

**Expected behavior**
The CVE and CVW codes should be included in the finding

**Deployment method** *(select with an `X`)*
- [x] Docker Compose
- [ ] Kubernetes
- [ ] GoDojo

**Environment information**
 - Operating System: Ubuntu 20.04  (Windows WSL)
 - DefectDojo version (see footer) or commit message: v. 2.11.0 ( c101e47b )

**Logs** 
n/a

**Sample scan files**
used existing DD appscan files 
tests/ibm_appscan_xml_file.xml
unittests/scans/ibm_app/testfire.xml

**Comment by StefanFl** [comment]
I will have a look at it.

**Comment by StefanFl** [comment]
Hi @mhanner, thanks for the patch. I don't understand the changes in the method `fetch_issue_types(self, root)`. Why are cve and cwe handled there as well and do you have a sample that shows where this is needed?

**Comment by mhanner** [comment]
Hi @StefanFl 

The current implementation does not extract the CWE or CVE values from the AppScan file:

![6479_original](https://user-images.githubusercontent.com/35705264/178326255-1f6864c3-24bb-482c-aacb-8d09965d18d1.jpg)

The patch will:
![6479_patched](https://user-images.githubusercontent.com/35705264/178326658-4396a9b1-57bb-4e7f-b79a-55a79660742d.jpg)


patch details:

re CVE
the original code calls:
_if "cve" in issue_type:
                        issues[issue_type.attrib['id']] = {'cve': issue_type.find("cve").text}_

The problem is cve is never added to issue_type so '_if "cve" in issue_type:_' always returns False.  The patch added cve to issue_type

re CWE
cwe is not extracted so I just added it in - cwe is added to issue_type.



FWIW, I found this while looking at [6035](https://github.com/DefectDojo/django-DefectDojo/issues/6035 ) (which I added a fix to).
